wiki_articles/vscode.md

```diff
@@ -0,0 +1,33 @@
+# Getting Started With VS Code for C/C++
+
+## DISCLAIMER FOR BEGINNERS
+Do not use VS Code.
+It's better to use an IDE that works out of the box:
+- [Visual Studio Community](https://visualstudio.microsoft.com/vs/community/) for Windows (free)
+- [CLion](https://www.jetbrains.com/clion/) on any platform (free for students)
+- [XCode](https://developer.apple.com/xcode/) on macOS (free)
+- [repl.it](https://replit.com/site/ide) online IDE (free for public projects)
+
+## 1. Install The C/C++ Extension by Microsoft
+Install [the extension](https://code.visualstudio.com/docs/languages/cpp) and
+follow the tutorial.
```

Comment:
The tutorial here starts with `Install a compiler`, so it might be good to move this section after the one below